src/main/java/uk/gov/hmcts/reform/civil/helpers/BundleRequestMapper.java

```diff
@@ -185,88 +186,101 @@ private List<Element<BundlingRequestDocument>> mapSystemGeneratedcaseDocument(Li
                 );
 
             });
-            if (null != orderSDODocumentDJ) {
-                bundlingSystemGeneratedCaseDocs.add(BundlingRequestDocument.builder()
-                                                        .documentFileName(orderSDODocumentDJ.getDocumentFileName())
-                                                        .documentLink(DocumentLink.builder()
-                                                                          .documentUrl(orderSDODocumentDJ.getDocumentUrl())
-                                                                          .documentBinaryUrl(orderSDODocumentDJ.getDocumentBinaryUrl())
-                                                                          .documentFilename(orderSDODocumentDJ.getDocumentFileName()).build())
-                                                        .documentType(DocumentType.SDO_ORDER.name()).build());
-            }
+        }
+        if (null != orderSDODocumentDJ) {
+            bundlingSystemGeneratedCaseDocs.add(BundlingRequestDocument.builder()
+                                                    .documentFileName(orderSDODocumentDJ.getDocumentFileName())
+                                                    .documentLink(DocumentLink.builder()
+                                                                      .documentUrl(orderSDODocumentDJ.getDocumentUrl())
+                                                                      .documentBinaryUrl(orderSDODocumentDJ.getDocumentBinaryUrl())
+                                                                      .documentFilename(orderSDODocumentDJ.getDocumentFileName()).build())
+                                                    .documentType(DocumentType.SDO_ORDER.name()).build());
```

Comment:
The "contract" of this ticket is simply to reduce complexity by leveraging short-circuiting, not altering the logic of the application.

Why has this been moved outside the "if" on line 174?

Reply:
This has been reverted and will be addressed in a separate ticket, should a problem arise.